File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.py
class Solution:
    def minimumLength(self, s: str) -> int:
        if len(s)<1:
            return 0
        if len(s)<2:
            return 1
        if len(s)<3:
            return 2
        
        l=len(s)
        c=Counter(s)
        x=0
        for key, value in c.items():
            while(value>=3):
                value-=2
            x+=value
        return x




        

File: 3455-minimum-length-of-string-after-operations/minimum-length-of-string-after-operations.cpp
class Solution {
public:
    int minimumLength(string s) {
        if(s.size()<1) return 0;
        if(s.size()<2) return 1;
        if(s.size()<3) return 2;
        int hash[26]={0};
        for(int i=0;i<s.size();i++){
            hash[s[i]-'a']++;
        }
        int x=0;
        for(int i=0;i<26;i++){
            while(hash[i]>=3){
                hash[i]-=2;
            }
            x+=hash[i];
        }
        return x;
    }
};